cinema/models.py

```diff
@@ -27,6 +27,10 @@ class Actor(models.Model):
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255)
 
+    @property
+    def full_name(self) -> str:
+        return self.first_name + " " + self.last_name
+
     def __str__(self):
         return self.first_name + " " + self.last_name
 
```

Comment:
I know that right now they are identical, but even in video Danylo said it's not good to rely on __str__ because it can be changed at any moment so that's why I decided to use "@property" here.